init.vim

```diff
@@ -225,6 +225,8 @@ let g:which_key_map = {
   \   't' : [':LspRestart', 'Restart LSP Server'],
   \   's' : [':LspStart', 'Start LSP Server'],
   \   'x' : [':LspStop', 'Stop LSP Server'],
+  \   'h' : [':call chansend(g:last_terminal_chan_id, ":r\<cr>")', 'Recompile in GHCI - send :r to the terminal'],
+  \   'p' : [':call chansend(g:last_terminal_chan_id, "grunt\<cr>")', 'Recompile Purescript - send grunt to the terminal'],
```

Comment:
I imagine we might want to find a way to make this less project-specific in the future, but I like the general idea.

Reply:
If we just built the PureScript container to alias `:r` (shell) to `grunt`, we'd need only one shortcut. PureScript is the outlier, that's why I suggest to do it that way.

Reply:
And then there would be no possibility of using the "wrong" shortcut, which I think is very likely with this PR as it currently is. What do you think @jbrechtel ?

Reply:
Yeah. I'd like to have a way to modify this map on a project by project basis - and even have user-specific files loaded. Maybe next PR.